common/aws/ecs/ecs.go

```diff
@@ -392,6 +392,12 @@ func (this *ECS) RunTask(cluster, taskDefinition string, count int64, startedBy
 		Count:          aws.Int64(count),
 		StartedBy:      startedBy,
 		Overrides:      taskOverride,
+		PlacementStrategy: []*ecs.PlacementStrategy{
```

Comment:
Huh... that's actually pretty weird. I think this is a somewhat unique "problem" since there are very few params parameters that an end user would potentially care about that _aren't_ represented in a task definition. 

To be clear, I feel like the documentation for this feature is incomplete; I'm still not totally sure what "default" is, even if we can make some educated guesses. But this is where we could make Layer0 a marked improvement over ECS in that we can expose these settings within our own configuration.

Anyway, here's ideas along those lines:
 
- We add some API params that allow clients to set the placementStrategy, either through CLI flags or terraform config
- We write some heuristics that make the placementStategy decision for the user 

Anyway, I'm OK with accepting this PR as is, but if I do, I think we should make another issue that allows configuring the placementStrategy.